ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -204,28 +225,46 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-shared-vpc-phz-freeipa-f28
-  cron: 40 1 14 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f28
+  cron: 32 19 16 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
     test:
-    - ref: idp-freeipa
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
-- as: aws-ipi-shared-vpc-phz-sts-fips-openldap-f28
-  cron: 53 0 6 * *
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: aws-ipi-proxy-sdn-fips-f28
+  cron: 24 22 26 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      EXTRACT_MANIFEST_INCLUDED: "true"
       FIPS_ENABLED: "true"
+      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
     test:
-    - ref: idp-openldap
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn
+- as: aws-ipi-shared-vpc-phz-f14
+  cron: 15 22 8,24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-wavelength-byo-subnet-ovn-day2-f28
```

Comment:
cc @yunjiang29 to help evaluate if need to cover this one, based on we already have the follow - `aws-ipi-wavelength-byo-subnet-ovn-mtu-day2` upgrade ci job.

Reply:
Agree,`aws-ipi-wavelength-byo-subnet-ovn` is more suitable, it contains newer installer feature.

Reply:
@jianlinliu we also have `aws-ipi-wavelength-byo-subnet-ovn-day2-f28` in `openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml` and `openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.17.yaml`. 
Do you think we'd better:
1. move `aws-ipi-wavelength-byo-subnet-ovn` from `openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml` ro `openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml`
2. move `openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.17.yaml` to `openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml`

Reply:
`aws-ipi-wavelength-byo-subnet-ovn` source e2e job exists in amd ci, so the upgrade job for the profile also should be synced to mult-arch upgrade ci.

Reply:
OK, right now there are:
```
[jianl@jianl-thinkpadt14gen4 release]$ grep -r 'aws-ipi-wavelength-byo-subnet-ovn' --include="*4.17-upgrade*"
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.17.yaml:- as: aws-ipi-wavelength-byo-subnet-ovn-f28
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:- as: aws-ipi-wavelength-byo-subnet-ovn-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
```